WTG.Analyzers.TestFramework/Helpers/DiagnosticUtils.cs

```diff
@@ -26,8 +27,22 @@ public static async Task<Diagnostic[]> GetDiagnosticsAsync(DiagnosticAnalyzer an
 
 			var diagnostics = new List<Diagnostic>();
 
-			foreach (var project in projects)
+			foreach (var p in projects)
 			{
+				var project = p;
+				if (project.ParseOptions.Language == LanguageNames.CSharp)
+				{
+					var parseOptions = ((CSharpParseOptions)project.ParseOptions)
+						.WithLanguageVersion(LanguageVersion.CSharp7_3);
```

Comment:
I would prefer we set the language version and 'allow unsafe' in [ModelUtils.AddProject()](https://github.com/WiseTechGlobal/WTG.Analyzers/blob/master/WTG.Analyzers.TestFramework/Helpers/ModelUtils.cs#L59) rather than here.